File: standalone/dzen2-hlwm/python/01-testparams.py
#!/usr/bin/env python3
# This is a modularized config for herbstluftwm tags in dzen2 statusbar

import sys
import helper

# ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----
# initialize

panel_height = 24
monitor = helper.get_monitor(sys.argv)

dzen2_parameters = helper.get_dzen2_parameters(monitor, panel_height)
print(dzen2_parameters)

File: standalone/dzen2-hlwm/ruby/01-testparams.rb
#!/usr/bin/ruby
# This is a modularized config for herbstluftwm tags in dzen2 statusbar

require_relative 'helper'
require_relative 'pipehandler'

# ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----- ----
# initialize

panel_height = 24
monitor = get_monitor(ARGV)

dzen2_parameters = get_dzen2_parameters(monitor, panel_height)
puts(dzen2_parameters)
